Source/Scene/Scene.js

```diff
@@ -505,6 +505,48 @@ define([
          */
         this.copyGlobeDepth = false;
 
+        /**
+         * Blends the atmosphere to geometry far from the camera for horizon views. Allows for additional
+         * performance improvements by rendering less geometry and dispatching less terrain requests.
+         * @type {Object}
+         */
+        this.fog = {
```

Comment:
I'm not sure about this doc this generates.  See how the `Type` and `Description` fields are empty:

![image](https://cloud.githubusercontent.com/assets/782098/10999579/ddbce626-846a-11e5-8bfc-ab93c3c4e549.png)

Perhaps we should just introduce a trivial `Fog` object to aid in documentation?